### DISCUSSION TITLE
When can the project add paging settings or scroll load files

### DISCUSSION THREAD

**Discussion by sunshinexiaozhu** [minio/minio#15368] [Q&A] [answered]
We are currently using Minio. During the use process, we found that we need to manually divide directories for storage. We encounter such a problem when viewing on the console. There are many files in a directory that will be loaded for a long time before the file list can be listed. Can we add the function of paging or scrolling to view files? At present, our main storage is pictures

**Comment by harshavardhana** [comment, marked as answer]
@sunshinexiaozhu do not dump files at the single prefix - it's a simple rule because your filesystem cannot handle beyond X amount of files per directory.

For a better experience choose a better hierarchy.
